golang/pre-publish/set-environment.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash
+set -eu
+
+COMMIT_TEMPLATE='BUMP v${VERSION}'
+TAG_VERSION="v${VERSION}"
+
+if [[ "${VERSION}" =~ cloud.* ]]; then
```

Comment:
Not sure if this is checked in the release trigger script, but left-anchoring the regex might help ensure we don't accidentally match more than we want:
```suggestion
if [[ "${VERSION}" =~ ^cloud.* ]]; then
```